ads/google/doubleclick.js

```diff
@@ -164,6 +172,12 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
   if (gladeExperiment === GladeExperiment.GLADE_EXPERIMENT) {
     jsonParameters.gladeExp = '1';
   }
+  const expIds = data['experimentId'];
+  if (expIds) {
+    jsonParameters.gladeEids = jsonParameters.gladeEids ?
+        jsonParameters.gladeEids + ',' + expIds : expIds;
```

Comment:
I assume that @mlb7687 has looked at this and verified that it looks good to him?